spec/spec_helper_acceptance.rb

```diff
@@ -1,18 +1,24 @@
 require 'beaker-rspec'
+require 'beaker/puppet_install_helper'
 require 'securerandom'
 require 'thread'
 require 'infrataster/rspec'
 require 'rspec/retry'
 require 'vault'
 
+require 'simp/beaker_helpers'
+include Simp::BeakerHelpers
+
 require_relative 'spec_helper_tls'
 require_relative 'spec_utilities'
 require_relative '../lib/puppet_x/elastic/deep_to_i'
 require_relative '../lib/puppet_x/elastic/deep_to_s'
 
-def f
-  RSpec.configuration.fact
-end
+# def f
+#   RSpec.configuration.fact
+# end
```

Comment:
can we remove this block if it isn't used?

Reply:
Yep, I'll clean these up in a future PR! :)